homeassistant/components/lyric/climate.py

```diff
@@ -152,6 +174,16 @@ def current_temperature(self) -> Optional[float]:
         """Return the current temperature."""
         return self.device.indoorTemperature
 
+    @property
+    def hvac_action(self) -> str:
+        """Return the current hvac action."""
+        action = HVAC_ACTIONS.get(
+            self.device.operationStatus.mode, self.device.operationStatus.mode
```

Comment:
We should handle the case where there isn't an HVAC_ACTIONS for the `self.device.operationStatus.mode` instead of returning `self.device.operationStatus.mode` as it may not be a valid action.

Reply:
Should `None` be the default here?

Reply:
If we don't have a valid one, return `None`